datamodel.customer-survey.xml

```diff
@@ -1922,6 +1925,64 @@ EOF
 				throw new Exception(Dict::S('Survey-email-notsent').': '.implode(', ', $aErrors));
 		}
 	}]]></code>
+        </method>
+        <method id="ApplyParamsToContent">
+          <comment>/**
+	 * @param string $sContent
+	 * @param Contact $oContact
+	 * @param ?string $sQuizzUrl
```

Comment:
Mind that in English it's actually a single Z :)

Reply:
Woops, thanks !
Fixed in be41bc6f